python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -101,7 +101,7 @@ def size(self):
                     len(self.obj), "int8", masked=False
                 )
             )
-            .groupby(self.grouping)
+            .groupby(self.grouping, sort=self._sort)
```

Comment:
I believe we should always use `sort=True` here.

Reply:
Sorry, I misunderstood  -- I don't think we need this, since the keys of the resulting `Series` hold the group keys.